cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -44,14 +42,10 @@ describe('[SMOKE] Create transactions tests', () => {
   })
 
   it('[SMOKE] Verify nonce tooltip warning messages', () => {
-    createtx.changeNonce(-1)
-    createtx.verifyTooltipMessage(constants.nonceTooltipMsg.lowerThanCurrent + currentNonce.toString())
     createtx.changeNonce(currentNonce + 50)
     createtx.verifyTooltipMessage(constants.nonceTooltipMsg.higherThanRecommended)
     createtx.changeNonce(currentNonce + 150)
     createtx.verifyTooltipMessage(constants.nonceTooltipMsg.muchHigherThanRecommended)
-    createtx.changeNonce('abc')
-    createtx.verifyTooltipMessage(constants.nonceTooltipMsg.mustBeNumber)
```

Comment:
The user can't enter such numbers anymore. 
anything with a decimal point would be valid though :(